core/matrix/dense.cpp

```diff
@@ -1010,7 +1029,6 @@ std::unique_ptr<Dense<ValueType>> Dense<ValueType>::row_gather(
     return result;
 }
 
-
```

Comment:
I believe this empty line should stay

Reply:
It is overloaded, so one empty is enough